compatimetrics_plots.py

```diff
@@ -0,0 +1,1029 @@
+import numpy as np
+import pandas as pd
+import plotly.express as px
+import plotly.graph_objects as go
+from metrics import empty_fig
+from compatimetrics import mean_squared_difference, \
+    root_mean_squared_difference, strong_disagreement_ratio, agreement_ratio, conjunctive_rmse, \
+    uniformity, disagreement_ratio, disagreement_postive_ratio, correctness_counter, \
+    conjunctive_accuracy, conjunctive_precission, conjunctive_recall, average_collective_score, \
+    macro_conjunctive_precission, weighted_conjunctive_precission, macro_conjunctive_recall, \
+    weighted_conjunctive_recall
+
+
+def msd_matrix(predictions):
+    """Mean Squared Difference matrix of every pair of models from the ensemble model.
+
+    Parameters
+    ----------
+    predictions: dictionary with predictions of ensemble component models
+        of form {'model_name': 'prediction_vector'}
+
+    Returns
+    -------
+    fig : plotly.graph_objs._figure.Figure
+        plotly plot
+
+    Examples
+    --------
+    msd_matrix(predictions)
+    """
+    models = list(predictions.keys())
+    n_models = len(models)
+    matrix = np.zeros((n_models, n_models))
+    for i in range(n_models):
+        for j in range(n_models):
+            if i == j:
+                matrix[i, j] = 0
+            else:
+                matrix[i, j] = round(mean_squared_difference(predictions[models[i]], predictions[models[j]]))
+    matrix = pd.DataFrame(matrix, index=models, columns=models)
+    custom_colors = ['rgba(242,26,155,255)',
+                     'rgba(254,113,0,255)',
+                     'rgba(255,168,0,255)',
+                     'rgba(125,179,67,255)',
+                     'rgba(3,169,245,255)']
+    fig = px.imshow(matrix, color_continuous_scale=custom_colors)
+    fig.update_layout(
+        title="Mean Squared Difference",
+        plot_bgcolor='rgba(44,47,56,255)',
+        paper_bgcolor='rgba(44,47,56,255)',
+        font_color="rgba(225, 225, 225, 255)",
+        font_size=17,
+        title_font_color="rgba(225, 225, 225, 255)",
+        title_font_size=20,
+        xaxis_title_standoff=300,
+        yaxis_ticklen=39,
+    )
```

Comment:
Instead of repeating this code with each plot, I would make a function that would change the layout